src/server/network/protocol/protocollogin.cpp

```diff
@@ -179,3 +191,56 @@ void ProtocolLogin::onRecvFirstMessage(NetworkMessage &msg) {
 	},
 							"ProtocolLogin::getCharacterList");
 }
+
+void ProtocolLogin::getCastViewersList(const std::string &password) {
+	auto output = OutputMessagePool::getOutputMessage();
+	output->addByte(0x14);
+	std::ostringstream ss;
+	ss << normal_random(1, 100) << "\nWelcome to Cast System!";
+	output->addString(ss.str(), "ProtocolLogin::getCastViewersList - ss.str()");
```

Comment:
This would have the same effect?
```suggestion
	output->addString(fmt::format("{}\nWelcome to Cast System!", normal_random(1, 100)), "ProtocolLogin::getCastViewersList - fmt::format('{}\nWelcome to Cast System!', normal_random(1, 100))");
```